test/unit/storage/conversion/QuadToRdfConverter.test.ts

```diff
@@ -57,6 +61,33 @@ describe('A QuadToRdfConverter', (): void => {
     );
   });
 
+  it('converts quads with prefixes to turtle.', async(): Promise<void> => {
+    metadata.addQuad(DC.terms(), PREFERRED_PREFIX_TERM, 'dc');
```

Comment:
As mentioned before I would prefer to never use these function calls to prevent misuse, so perhaps we could have a `DC.term` that corresponds to the base term? (And perhaps use fancy toString magic to have `DC` correspond to the base URI string but less necessary).

Reply:
Wouldn't do the fancy `toString` magic; will indeed remove function and expose `namespace` instead.